kidviz/views.py

```diff
@@ -156,45 +174,32 @@ def get(self, request, *args, **kwargs):
         return super().get(request, *args, **kwargs)
 
     def post(self, request, *args, **kwargs):
-        if self.request.POST.get('use_recent_observation'):
+        if request.POST.get('use_recent_observation'):
             return self.get(request, *args, **kwargs)
         else:
             draft_observation = Observation.objects.filter(is_draft=True).order_by('-id').first()
+            original_image = request.POST.get('original_image', None)
+            video = request.POST.get('video', None)
 
-            # This is needed to update video or original_image when one of them is already set
-            # and user want to change to the opposite.
-            if draft_observation:
-                if draft_observation.video and self.request.POST.get('original_image', None):
-                    draft_observation.video = None
-                    draft_observation.save()
-
-                if draft_observation.original_image and self.request.POST.get('video', None):
-                    draft_observation.original_image = None
-                    draft_observation.save()
+            update_draft_media(draft_observation, original_image, video)
 
-            if self.request.POST.get('is_draft', 'False') == 'True':
+            if request.POST.get('is_draft', 'False') == 'True':
```

Comment:
I know this is an old code, but imo it looks strange. Why can't we just make `request.POST.get('is_draft', False) = 'True'`. Or even None instead of False. Or event nothing, just `get('is_draft)`. Correct me if I wrong. Get returns `None` if there is no key in dict.